src/signals/incident/components/form/HandlingMessage/style.scss

```diff
@@ -0,0 +1,29 @@
+.handling-message {
+  margin: 16px 0;
+
+  &__box {
```

Comment:
Since we aren't adhering to the entire BEM philosophy, we can let go of BEM naming format.